src/Factory/DelegatingNonceFactory.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+namespace RebelCode\WordPress\Nonce\Factory;
+
+use Dhii\Factory\FactoryInterface;
+
+/**
+ * A nonce factory implementation that delegates to another generic factory.
+ *
+ * @since [*next-version*]
+ */
+class DelegatingNonceFactory extends AbstractNonceFactory implements FactoryInterface
+{
+    /**
+     * The config array key to use to pass the nonce code to the delegate factory.
+     *
+     * @since [*next-version*]
+     */
+    const K_CONFIG_ID = 'id';
+
+    /**
+     * The config array key to use to pass the nonce code to the delegate factory.
+     *
+     * @since [*next-version*]
+     */
+    const K_CONFIG_CODE = 'code';
+
+    /**
+     * The default ID used by the delegate factory for nonces.
+     *
+     * @since [*next-version*]
+     */
+    const DEFAULT_DELEGATE_ID = 'nonce';
+
+    /**
+     * The factory instance to which nonce instance creation will be delegated to.
+     *
+     * @since [*next-version*]
+     *
+     * @var FactoryInterface
+     */
+    protected $delegateFactory;
+
+    /**
+     * The ID for nonces in the delegate factory.
+     *
+     * @since [*next-version*]
+     *
+     * @var string
+     */
+    protected $delegateId;
+
+    /**
+     * Constructor.
+     *
+     * @since [*next-version*]
+     *
+     * @param FactoryInterface $factory    The factory to which to delegate nonce creation to.
+     * @param string           $delegateId The ID for nonces in the delegate factory.
+     */
+    public function __construct(FactoryInterface $factory, $delegateId = self::DEFAULT_DELEGATE_ID)
+    {
+        $this->_setDelegateFactory($factory)
+            ->_setDelegateId($delegateId);
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @since [*next-version*]
+     */
+    public function make($id, array $config = [])
+    {
+        return $this->_make($id, $config);
```

Comment:
The `_make` method is expected to create the entire nonce, that is, its data and the instance for that data. The `_createNonce` method is expected to create only the instance with given data.